eoscostarica/eos-rate
# Refactor/smart-contract

### GH Issue

Resolves #588 

Separate logic into the main folders `include`  and `src` and also create a script folder to which contains the needed logic to make a faster Smart Contract compilation

### Steps to test

1. Go to `~/eos-rate/contracts/scripts`
1. Run `./compile.sh`
1. Check Compilation process is correct
